data.sql

```diff
@@ -5,4 +5,10 @@ VALUES
 (1, 'Agumon', '2020-2-3', 0, true, 10.23),
 (2, 'Gabumon', '2018-11-5', 2, true, 8),
 (3, 'Pikachu', '2021-1-7', 1, false, 15.4),
-(4, 'Devimon', '2017-5-12', 5, true, 11);
\ No newline at end of file
+(4, 'Devimon', '2017-5-12', 5, true, 11),
+(5, 'Charmander', '2020-02-08', 0, false, -11),
+(6, 'Plantmon', '2022-11-15', 2, TRUE, -5.7),
+(7, 'Squirtle', '1993-04-02', 3, FALSE, -12.13),
+(8, 'Angemon', '2005-06-12', 1, TRUE, -45),
+(9, 'Boarmon', '2005-06-07', 7, TRUE, 20.4),
+(10, 'Blossom', '1998-10-13', 3, TRUE, 17);
```

Comment:
Kindly insert this data ` Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times. ` it is missing [Refer here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Animal%3A%20His%20name%20is%20Ditto.%20He%20was%20born%20on%20May%2014th%2C%202022%2C%20and%20currently%20weighs%2022kg.%20He%20is%20neutered%20and%20he%20has%20tried%20to%20escape%204%20times.)